File: src/codes/Users_Manual/stm_prop.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())}).open(autd3::link::Nop::builder());
std::vector<std::array<autd3::Vector3, 1>> foci = {
    std::array{autd3::Vector3(0, 0, 0)},
    std::array{autd3::Vector3(0, 0, 0)},
};
const auto stm = autd3::FociSTM<1>::from_freq(1.0f * autd3::Hz, foci);
const auto f = stm.freq();                     // -> 1Hz
const auto p = stm.period();                   // -> 1s
const auto fs = stm.sampling_config().freq();  // -> 2Hz
                                               //~return 0; }

File: src/codes/Users_Manual/stm_prop.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Link;
~using static AUTD3Sharp.Units;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
var stm = FociSTM.FromSamplingConfig(1u * Hz, [Vector3.Zero, Vector3.Zero]);
var f = stm.Freq;                 // -> 1Hz
var p = stm.Period;               // -> 1s
var fs = stm.SamplingConfig.Freq; // -> 2Hz